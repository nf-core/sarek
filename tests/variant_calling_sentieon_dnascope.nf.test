nextflow_pipeline {

    name "Test pipeline"
    script "../main.nf"
    tag "pipeline"
    tag "pipeline_sarek"
    tag "cpu"

    def test_scenario = [
        [
            name: "-profile test --tools sentieon_dnascope",
            params: [
                input: "${projectDir}/tests/csv/3.0/mapped_single_bam.csv",
                outdir: "$outputDir",
                tools: "sentieon_dnascope",
                step: "variant_calling"
            ]
        ],
        [
            name: "-profile test --tools sentieon_dnascope --skip_tools dnascope_filter",
            params: [
                input: "${projectDir}/tests/csv/3.0/mapped_single_bam.csv",
                outdir: "$outputDir",
                tools: "sentieon_dnascope",
                step: "variant_calling",
                skip_tools: "dnascope_filter"
            ]
        ],
        [
            name: "-profile test --tools sentieon_dnascope --joint_germline --sentieon_dnascope_emit_mode gvcf",
            params: [
                input: "${projectDir}/tests/csv/3.0/mapped_single_bam.csv",
                outdir: "$outputDir",
                tools: "sentieon_dnascope",
                step: "variant_calling",
                joint_germline: true,
                sentieon_dnascope_emit_mode: "gvcf"
            ]
        ]
    ]

    test_scenario.each { scenario ->
        test(scenario.name, UTILS.get_test(scenario))
    }
}
