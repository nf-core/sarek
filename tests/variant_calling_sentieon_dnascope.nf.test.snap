{
    "-profile test --tools sentieon_dnascope --skip_tools dnascope_filter": {
        "content": [
            12,
            {
                "BCFTOOLS_STATS": {
                    "bcftools": 1.21
                },
                "CREATE_INTERVALS_BED": {
                    "gawk": "5.3.0"
                },
                "GATK4_INTERVALLISTTOBED": {
                    "gatk4": "4.6.1.0"
                },
                "MOSDEPTH": {
                    "mosdepth": "0.3.10"
                },
                "SAMTOOLS_STATS": {
                    "samtools": 1.21
                },
                "SENTIEON_DNASCOPE": {
                    "sentieon": 202503.01
                },
                "TABIX_BGZIPTABIX_INTERVAL_COMBINED": {
                    "tabix": 1.21
                },
                "TABIX_BGZIPTABIX_INTERVAL_SPLIT": {
                    "tabix": 1.21
                },
                "VCFTOOLS_TSTV_COUNT": {
                    "vcftools": "0.1.16"
                }
            },
            [
                "csv",
                "csv/variantcalled.csv",
                "multiqc",
                "multiqc/multiqc_data",
                "multiqc/multiqc_data/BETA-multiqc.parquet",
                "multiqc/multiqc_data/bcftools-stats-subtypes.txt",
                "multiqc/multiqc_data/bcftools_stats_indel-lengths.txt",
                "multiqc/multiqc_data/bcftools_stats_variant_depths.txt",
                "multiqc/multiqc_data/bcftools_stats_vqc_Count_Indels.txt",
                "multiqc/multiqc_data/bcftools_stats_vqc_Count_SNP.txt",
                "multiqc/multiqc_data/bcftools_stats_vqc_Count_Transitions.txt",
                "multiqc/multiqc_data/bcftools_stats_vqc_Count_Transversions.txt",
                "multiqc/multiqc_data/llms-full.txt",
                "multiqc/multiqc_data/mosdepth-coverage-per-contig-single.txt",
                "multiqc/multiqc_data/mosdepth-cumcoverage-dist-id.txt",
                "multiqc/multiqc_data/mosdepth_cov_dist.txt",
                "multiqc/multiqc_data/mosdepth_cumcov_dist.txt",
                "multiqc/multiqc_data/mosdepth_perchrom.txt",
                "multiqc/multiqc_data/multiqc.log",
                "multiqc/multiqc_data/multiqc_bcftools_stats.txt",
                "multiqc/multiqc_data/multiqc_citations.txt",
                "multiqc/multiqc_data/multiqc_data.json",
                "multiqc/multiqc_data/multiqc_general_stats.txt",
                "multiqc/multiqc_data/multiqc_samtools_stats.txt",
                "multiqc/multiqc_data/multiqc_software_versions.txt",
                "multiqc/multiqc_data/multiqc_sources.txt",
                "multiqc/multiqc_data/samtools-stats-dp.txt",
                "multiqc/multiqc_data/samtools_alignment_plot.txt",
                "multiqc/multiqc_data/vcftools_tstv_by_count.txt",
                "multiqc/multiqc_data/vcftools_tstv_by_qual.txt",
                "multiqc/multiqc_plots",
                "multiqc/multiqc_plots/pdf",
                "multiqc/multiqc_plots/pdf/bcftools-stats-subtypes-cnt.pdf",
                "multiqc/multiqc_plots/pdf/bcftools-stats-subtypes-pct.pdf",
                "multiqc/multiqc_plots/pdf/bcftools_stats_indel-lengths-cnt.pdf",
                "multiqc/multiqc_plots/pdf/bcftools_stats_indel-lengths-log.pdf",
                "multiqc/multiqc_plots/pdf/bcftools_stats_variant_depths.pdf",
                "multiqc/multiqc_plots/pdf/bcftools_stats_vqc_Count_Indels.pdf",
                "multiqc/multiqc_plots/pdf/bcftools_stats_vqc_Count_SNP.pdf",
                "multiqc/multiqc_plots/pdf/bcftools_stats_vqc_Count_Transitions.pdf",
                "multiqc/multiqc_plots/pdf/bcftools_stats_vqc_Count_Transversions.pdf",
                "multiqc/multiqc_plots/pdf/mosdepth-coverage-per-contig-single-cnt.pdf",
                "multiqc/multiqc_plots/pdf/mosdepth-coverage-per-contig-single-pct.pdf",
                "multiqc/multiqc_plots/pdf/mosdepth-cumcoverage-dist-id.pdf",
                "multiqc/multiqc_plots/pdf/samtools-stats-dp.pdf",
                "multiqc/multiqc_plots/pdf/samtools_alignment_plot-cnt.pdf",
                "multiqc/multiqc_plots/pdf/samtools_alignment_plot-pct.pdf",
                "multiqc/multiqc_plots/pdf/vcftools_tstv_by_count.pdf",
                "multiqc/multiqc_plots/pdf/vcftools_tstv_by_qual.pdf",
                "multiqc/multiqc_plots/png",
                "multiqc/multiqc_plots/png/bcftools-stats-subtypes-cnt.png",
                "multiqc/multiqc_plots/png/bcftools-stats-subtypes-pct.png",
                "multiqc/multiqc_plots/png/bcftools_stats_indel-lengths-cnt.png",
                "multiqc/multiqc_plots/png/bcftools_stats_indel-lengths-log.png",
                "multiqc/multiqc_plots/png/bcftools_stats_variant_depths.png",
                "multiqc/multiqc_plots/png/bcftools_stats_vqc_Count_Indels.png",
                "multiqc/multiqc_plots/png/bcftools_stats_vqc_Count_SNP.png",
                "multiqc/multiqc_plots/png/bcftools_stats_vqc_Count_Transitions.png",
                "multiqc/multiqc_plots/png/bcftools_stats_vqc_Count_Transversions.png",
                "multiqc/multiqc_plots/png/mosdepth-coverage-per-contig-single-cnt.png",
                "multiqc/multiqc_plots/png/mosdepth-coverage-per-contig-single-pct.png",
                "multiqc/multiqc_plots/png/mosdepth-cumcoverage-dist-id.png",
                "multiqc/multiqc_plots/png/samtools-stats-dp.png",
                "multiqc/multiqc_plots/png/samtools_alignment_plot-cnt.png",
                "multiqc/multiqc_plots/png/samtools_alignment_plot-pct.png",
                "multiqc/multiqc_plots/png/vcftools_tstv_by_count.png",
                "multiqc/multiqc_plots/png/vcftools_tstv_by_qual.png",
                "multiqc/multiqc_plots/svg",
                "multiqc/multiqc_plots/svg/bcftools-stats-subtypes-cnt.svg",
                "multiqc/multiqc_plots/svg/bcftools-stats-subtypes-pct.svg",
                "multiqc/multiqc_plots/svg/bcftools_stats_indel-lengths-cnt.svg",
                "multiqc/multiqc_plots/svg/bcftools_stats_indel-lengths-log.svg",
                "multiqc/multiqc_plots/svg/bcftools_stats_variant_depths.svg",
                "multiqc/multiqc_plots/svg/bcftools_stats_vqc_Count_Indels.svg",
                "multiqc/multiqc_plots/svg/bcftools_stats_vqc_Count_SNP.svg",
                "multiqc/multiqc_plots/svg/bcftools_stats_vqc_Count_Transitions.svg",
                "multiqc/multiqc_plots/svg/bcftools_stats_vqc_Count_Transversions.svg",
                "multiqc/multiqc_plots/svg/mosdepth-coverage-per-contig-single-cnt.svg",
                "multiqc/multiqc_plots/svg/mosdepth-coverage-per-contig-single-pct.svg",
                "multiqc/multiqc_plots/svg/mosdepth-cumcoverage-dist-id.svg",
                "multiqc/multiqc_plots/svg/samtools-stats-dp.svg",
                "multiqc/multiqc_plots/svg/samtools_alignment_plot-cnt.svg",
                "multiqc/multiqc_plots/svg/samtools_alignment_plot-pct.svg",
                "multiqc/multiqc_plots/svg/vcftools_tstv_by_count.svg",
                "multiqc/multiqc_plots/svg/vcftools_tstv_by_qual.svg",
                "multiqc/multiqc_report.html",
                "pipeline_info",
                "pipeline_info/nf_core_sarek_software_mqc_versions.yml",
                "reference",
                "reports",
                "reports/bcftools",
                "reports/bcftools/sentieon_dnascope",
                "reports/bcftools/sentieon_dnascope/test",
                "reports/bcftools/sentieon_dnascope/test/test.dnascope.unfiltered.bcftools_stats.txt",
                "reports/mosdepth",
                "reports/mosdepth/test",
                "reports/mosdepth/test/test.recal.mosdepth.global.dist.txt",
                "reports/mosdepth/test/test.recal.mosdepth.region.dist.txt",
                "reports/mosdepth/test/test.recal.mosdepth.summary.txt",
                "reports/mosdepth/test/test.recal.regions.bed.gz",
                "reports/mosdepth/test/test.recal.regions.bed.gz.csi",
                "reports/samtools",
                "reports/samtools/test",
                "reports/samtools/test/test.recal.cram.stats",
                "reports/vcftools",
                "reports/vcftools/sentieon_dnascope",
                "reports/vcftools/sentieon_dnascope/test",
                "reports/vcftools/sentieon_dnascope/test/test.dnascope.unfiltered.FILTER.summary",
                "reports/vcftools/sentieon_dnascope/test/test.dnascope.unfiltered.TsTv.count",
                "reports/vcftools/sentieon_dnascope/test/test.dnascope.unfiltered.TsTv.qual",
                "variant_calling",
                "variant_calling/sentieon_dnascope",
                "variant_calling/sentieon_dnascope/test",
                "variant_calling/sentieon_dnascope/test/test.dnascope.unfiltered.vcf.gz",
                "variant_calling/sentieon_dnascope/test/test.dnascope.unfiltered.vcf.gz.tbi"
            ],
            [
                "mosdepth-coverage-per-contig-single.txt:md5,115b9a93bcf511419e6622919172f321",
                "mosdepth-cumcoverage-dist-id.txt:md5,df2fc825dfc41e30f72f5d125a6447ee",
                "mosdepth_perchrom.txt:md5,115b9a93bcf511419e6622919172f321",
                "multiqc_citations.txt:md5,d40980f61eb64026d58102841b7f3860",
                "test.dnascope.unfiltered.bcftools_stats.txt:md5,ccd33b2c34ef09efe1fd7a86474eaa53",
                "test.dnascope.unfiltered.FILTER.summary:md5,87a84b5f8ac3d3cbeeef7d60afcdbfe7",
                "test.dnascope.unfiltered.TsTv.count:md5,b77c120ee5cc0423267200c67d60c663"
            ],
            "No BAM files",
            "No CRAM files",
            [
                "test.dnascope.unfiltered.vcf.gz:md5,372f2f8f6588b98ceaaae8ecd21d4f6e"
            ]
        ],
        "meta": {
            "nf-test": "0.9.2",
            "nextflow": "25.04.2"
        },
        "timestamp": "2025-06-21T12:35:10.846318732"
    },
    "-profile test --tools sentieon_dnascope": {
        "content": [
            13,
            {
                "BCFTOOLS_STATS": {
                    "bcftools": 1.21
                },
                "CREATE_INTERVALS_BED": {
                    "gawk": "5.3.0"
                },
                "GATK4_INTERVALLISTTOBED": {
                    "gatk4": "4.6.1.0"
                },
                "MOSDEPTH": {
                    "mosdepth": "0.3.10"
                },
                "SAMTOOLS_STATS": {
                    "samtools": 1.21
                },
                "SENTIEON_DNAMODELAPPLY": {
                    "sentieon": 202503.01
                },
                "SENTIEON_DNASCOPE": {
                    "sentieon": 202503.01
                },
                "TABIX_BGZIPTABIX_INTERVAL_COMBINED": {
                    "tabix": 1.21
                },
                "TABIX_BGZIPTABIX_INTERVAL_SPLIT": {
                    "tabix": 1.21
                },
                "VCFTOOLS_TSTV_COUNT": {
                    "vcftools": "0.1.16"
                }
            },
            [
                "csv",
                "csv/variantcalled.csv",
                "multiqc",
                "multiqc/multiqc_data",
                "multiqc/multiqc_data/BETA-multiqc.parquet",
                "multiqc/multiqc_data/bcftools-stats-subtypes.txt",
                "multiqc/multiqc_data/bcftools_stats_indel-lengths.txt",
                "multiqc/multiqc_data/bcftools_stats_variant_depths.txt",
                "multiqc/multiqc_data/bcftools_stats_vqc_Count_Indels.txt",
                "multiqc/multiqc_data/bcftools_stats_vqc_Count_SNP.txt",
                "multiqc/multiqc_data/bcftools_stats_vqc_Count_Transitions.txt",
                "multiqc/multiqc_data/bcftools_stats_vqc_Count_Transversions.txt",
                "multiqc/multiqc_data/llms-full.txt",
                "multiqc/multiqc_data/mosdepth-coverage-per-contig-single.txt",
                "multiqc/multiqc_data/mosdepth-cumcoverage-dist-id.txt",
                "multiqc/multiqc_data/mosdepth_cov_dist.txt",
                "multiqc/multiqc_data/mosdepth_cumcov_dist.txt",
                "multiqc/multiqc_data/mosdepth_perchrom.txt",
                "multiqc/multiqc_data/multiqc.log",
                "multiqc/multiqc_data/multiqc_bcftools_stats.txt",
                "multiqc/multiqc_data/multiqc_citations.txt",
                "multiqc/multiqc_data/multiqc_data.json",
                "multiqc/multiqc_data/multiqc_general_stats.txt",
                "multiqc/multiqc_data/multiqc_samtools_stats.txt",
                "multiqc/multiqc_data/multiqc_software_versions.txt",
                "multiqc/multiqc_data/multiqc_sources.txt",
                "multiqc/multiqc_data/samtools-stats-dp.txt",
                "multiqc/multiqc_data/samtools_alignment_plot.txt",
                "multiqc/multiqc_data/vcftools_tstv_by_count.txt",
                "multiqc/multiqc_data/vcftools_tstv_by_qual.txt",
                "multiqc/multiqc_plots",
                "multiqc/multiqc_plots/pdf",
                "multiqc/multiqc_plots/pdf/bcftools-stats-subtypes-cnt.pdf",
                "multiqc/multiqc_plots/pdf/bcftools-stats-subtypes-pct.pdf",
                "multiqc/multiqc_plots/pdf/bcftools_stats_indel-lengths-cnt.pdf",
                "multiqc/multiqc_plots/pdf/bcftools_stats_indel-lengths-log.pdf",
                "multiqc/multiqc_plots/pdf/bcftools_stats_variant_depths.pdf",
                "multiqc/multiqc_plots/pdf/bcftools_stats_vqc_Count_Indels.pdf",
                "multiqc/multiqc_plots/pdf/bcftools_stats_vqc_Count_SNP.pdf",
                "multiqc/multiqc_plots/pdf/bcftools_stats_vqc_Count_Transitions.pdf",
                "multiqc/multiqc_plots/pdf/bcftools_stats_vqc_Count_Transversions.pdf",
                "multiqc/multiqc_plots/pdf/mosdepth-coverage-per-contig-single-cnt.pdf",
                "multiqc/multiqc_plots/pdf/mosdepth-coverage-per-contig-single-pct.pdf",
                "multiqc/multiqc_plots/pdf/mosdepth-cumcoverage-dist-id.pdf",
                "multiqc/multiqc_plots/pdf/samtools-stats-dp.pdf",
                "multiqc/multiqc_plots/pdf/samtools_alignment_plot-cnt.pdf",
                "multiqc/multiqc_plots/pdf/samtools_alignment_plot-pct.pdf",
                "multiqc/multiqc_plots/pdf/vcftools_tstv_by_count.pdf",
                "multiqc/multiqc_plots/pdf/vcftools_tstv_by_qual.pdf",
                "multiqc/multiqc_plots/png",
                "multiqc/multiqc_plots/png/bcftools-stats-subtypes-cnt.png",
                "multiqc/multiqc_plots/png/bcftools-stats-subtypes-pct.png",
                "multiqc/multiqc_plots/png/bcftools_stats_indel-lengths-cnt.png",
                "multiqc/multiqc_plots/png/bcftools_stats_indel-lengths-log.png",
                "multiqc/multiqc_plots/png/bcftools_stats_variant_depths.png",
                "multiqc/multiqc_plots/png/bcftools_stats_vqc_Count_Indels.png",
                "multiqc/multiqc_plots/png/bcftools_stats_vqc_Count_SNP.png",
                "multiqc/multiqc_plots/png/bcftools_stats_vqc_Count_Transitions.png",
                "multiqc/multiqc_plots/png/bcftools_stats_vqc_Count_Transversions.png",
                "multiqc/multiqc_plots/png/mosdepth-coverage-per-contig-single-cnt.png",
                "multiqc/multiqc_plots/png/mosdepth-coverage-per-contig-single-pct.png",
                "multiqc/multiqc_plots/png/mosdepth-cumcoverage-dist-id.png",
                "multiqc/multiqc_plots/png/samtools-stats-dp.png",
                "multiqc/multiqc_plots/png/samtools_alignment_plot-cnt.png",
                "multiqc/multiqc_plots/png/samtools_alignment_plot-pct.png",
                "multiqc/multiqc_plots/png/vcftools_tstv_by_count.png",
                "multiqc/multiqc_plots/png/vcftools_tstv_by_qual.png",
                "multiqc/multiqc_plots/svg",
                "multiqc/multiqc_plots/svg/bcftools-stats-subtypes-cnt.svg",
                "multiqc/multiqc_plots/svg/bcftools-stats-subtypes-pct.svg",
                "multiqc/multiqc_plots/svg/bcftools_stats_indel-lengths-cnt.svg",
                "multiqc/multiqc_plots/svg/bcftools_stats_indel-lengths-log.svg",
                "multiqc/multiqc_plots/svg/bcftools_stats_variant_depths.svg",
                "multiqc/multiqc_plots/svg/bcftools_stats_vqc_Count_Indels.svg",
                "multiqc/multiqc_plots/svg/bcftools_stats_vqc_Count_SNP.svg",
                "multiqc/multiqc_plots/svg/bcftools_stats_vqc_Count_Transitions.svg",
                "multiqc/multiqc_plots/svg/bcftools_stats_vqc_Count_Transversions.svg",
                "multiqc/multiqc_plots/svg/mosdepth-coverage-per-contig-single-cnt.svg",
                "multiqc/multiqc_plots/svg/mosdepth-coverage-per-contig-single-pct.svg",
                "multiqc/multiqc_plots/svg/mosdepth-cumcoverage-dist-id.svg",
                "multiqc/multiqc_plots/svg/samtools-stats-dp.svg",
                "multiqc/multiqc_plots/svg/samtools_alignment_plot-cnt.svg",
                "multiqc/multiqc_plots/svg/samtools_alignment_plot-pct.svg",
                "multiqc/multiqc_plots/svg/vcftools_tstv_by_count.svg",
                "multiqc/multiqc_plots/svg/vcftools_tstv_by_qual.svg",
                "multiqc/multiqc_report.html",
                "pipeline_info",
                "pipeline_info/nf_core_sarek_software_mqc_versions.yml",
                "reference",
                "reports",
                "reports/bcftools",
                "reports/bcftools/sentieon_dnascope",
                "reports/bcftools/sentieon_dnascope/test",
                "reports/bcftools/sentieon_dnascope/test/test.dnascope.filtered.bcftools_stats.txt",
                "reports/mosdepth",
                "reports/mosdepth/test",
                "reports/mosdepth/test/test.recal.mosdepth.global.dist.txt",
                "reports/mosdepth/test/test.recal.mosdepth.region.dist.txt",
                "reports/mosdepth/test/test.recal.mosdepth.summary.txt",
                "reports/mosdepth/test/test.recal.regions.bed.gz",
                "reports/mosdepth/test/test.recal.regions.bed.gz.csi",
                "reports/samtools",
                "reports/samtools/test",
                "reports/samtools/test/test.recal.cram.stats",
                "reports/vcftools",
                "reports/vcftools/sentieon_dnascope",
                "reports/vcftools/sentieon_dnascope/test",
                "reports/vcftools/sentieon_dnascope/test/test.dnascope.filtered.FILTER.summary",
                "reports/vcftools/sentieon_dnascope/test/test.dnascope.filtered.TsTv.count",
                "reports/vcftools/sentieon_dnascope/test/test.dnascope.filtered.TsTv.qual",
                "variant_calling",
                "variant_calling/sentieon_dnascope",
                "variant_calling/sentieon_dnascope/test",
                "variant_calling/sentieon_dnascope/test/test.dnascope.filtered.vcf.gz",
                "variant_calling/sentieon_dnascope/test/test.dnascope.filtered.vcf.gz.tbi",
                "variant_calling/sentieon_dnascope/test/test.dnascope.unfiltered.vcf.gz",
                "variant_calling/sentieon_dnascope/test/test.dnascope.unfiltered.vcf.gz.tbi"
            ],
            [
                "mosdepth-coverage-per-contig-single.txt:md5,115b9a93bcf511419e6622919172f321",
                "mosdepth-cumcoverage-dist-id.txt:md5,df2fc825dfc41e30f72f5d125a6447ee",
                "mosdepth_perchrom.txt:md5,115b9a93bcf511419e6622919172f321",
                "multiqc_citations.txt:md5,d40980f61eb64026d58102841b7f3860",
                "test.dnascope.filtered.bcftools_stats.txt:md5,ad4472e0b4767e0f3c4052ff3e3c1cdb",
                "test.dnascope.filtered.FILTER.summary:md5,e67b24d296810a075378e5864bcea0fa",
                "test.dnascope.filtered.TsTv.count:md5,b77c120ee5cc0423267200c67d60c663"
            ],
            "No BAM files",
            "No CRAM files",
            [
                "test.dnascope.filtered.vcf.gz:md5,3415f9c0cbc1492008b70febe02f3e3e",
                "test.dnascope.unfiltered.vcf.gz:md5,372f2f8f6588b98ceaaae8ecd21d4f6e"
            ]
        ],
        "meta": {
            "nf-test": "0.9.2",
            "nextflow": "25.04.2"
        },
        "timestamp": "2025-06-21T11:44:42.944302645"
    },
    "-profile test --tools sentieon_dnascope --joint_germline --sentieon_dnascope_emit_mode gvcf": {
        "content": [
            15,
            {
                "BCFTOOLS_STATS": {
                    "bcftools": 1.21
                },
                "CREATE_INTERVALS_BED": {
                    "gawk": "5.3.0"
                },
                "GATK4_INTERVALLISTTOBED": {
                    "gatk4": "4.6.1.0"
                },
                "MOSDEPTH": {
                    "mosdepth": "0.3.10"
                },
                "SAMTOOLS_STATS": {
                    "samtools": 1.21
                },
                "SENTIEON_DNASCOPE": {
                    "sentieon": 202503.01
                },
                "SENTIEON_GVCFTYPER": {
                    "sentieon": 202503.01
                },
                "TABIX_BGZIPTABIX_INTERVAL_COMBINED": {
                    "tabix": 1.21
                },
                "TABIX_BGZIPTABIX_INTERVAL_SPLIT": {
                    "tabix": 1.21
                },
                "VCFTOOLS_TSTV_COUNT": {
                    "vcftools": "0.1.16"
                }
            },
            [
                "csv",
                "csv/variantcalled.csv",
                "multiqc",
                "multiqc/multiqc_data",
                "multiqc/multiqc_data/BETA-multiqc.parquet",
                "multiqc/multiqc_data/bcftools-stats-subtypes.txt",
                "multiqc/multiqc_data/bcftools_stats_indel-lengths.txt",
                "multiqc/multiqc_data/bcftools_stats_variant_depths.txt",
                "multiqc/multiqc_data/bcftools_stats_vqc_Count_Indels.txt",
                "multiqc/multiqc_data/bcftools_stats_vqc_Count_SNP.txt",
                "multiqc/multiqc_data/bcftools_stats_vqc_Count_Transitions.txt",
                "multiqc/multiqc_data/bcftools_stats_vqc_Count_Transversions.txt",
                "multiqc/multiqc_data/llms-full.txt",
                "multiqc/multiqc_data/mosdepth-coverage-per-contig-single.txt",
                "multiqc/multiqc_data/mosdepth-cumcoverage-dist-id.txt",
                "multiqc/multiqc_data/mosdepth_cov_dist.txt",
                "multiqc/multiqc_data/mosdepth_cumcov_dist.txt",
                "multiqc/multiqc_data/mosdepth_perchrom.txt",
                "multiqc/multiqc_data/multiqc.log",
                "multiqc/multiqc_data/multiqc_bcftools_stats.txt",
                "multiqc/multiqc_data/multiqc_citations.txt",
                "multiqc/multiqc_data/multiqc_data.json",
                "multiqc/multiqc_data/multiqc_general_stats.txt",
                "multiqc/multiqc_data/multiqc_samtools_stats.txt",
                "multiqc/multiqc_data/multiqc_software_versions.txt",
                "multiqc/multiqc_data/multiqc_sources.txt",
                "multiqc/multiqc_data/samtools-stats-dp.txt",
                "multiqc/multiqc_data/samtools_alignment_plot.txt",
                "multiqc/multiqc_data/vcftools_tstv_by_count.txt",
                "multiqc/multiqc_data/vcftools_tstv_by_qual.txt",
                "multiqc/multiqc_plots",
                "multiqc/multiqc_plots/pdf",
                "multiqc/multiqc_plots/pdf/bcftools-stats-subtypes-cnt.pdf",
                "multiqc/multiqc_plots/pdf/bcftools-stats-subtypes-pct.pdf",
                "multiqc/multiqc_plots/pdf/bcftools_stats_indel-lengths-cnt.pdf",
                "multiqc/multiqc_plots/pdf/bcftools_stats_indel-lengths-log.pdf",
                "multiqc/multiqc_plots/pdf/bcftools_stats_variant_depths.pdf",
                "multiqc/multiqc_plots/pdf/bcftools_stats_vqc_Count_Indels.pdf",
                "multiqc/multiqc_plots/pdf/bcftools_stats_vqc_Count_SNP.pdf",
                "multiqc/multiqc_plots/pdf/bcftools_stats_vqc_Count_Transitions.pdf",
                "multiqc/multiqc_plots/pdf/bcftools_stats_vqc_Count_Transversions.pdf",
                "multiqc/multiqc_plots/pdf/mosdepth-coverage-per-contig-single-cnt.pdf",
                "multiqc/multiqc_plots/pdf/mosdepth-coverage-per-contig-single-pct.pdf",
                "multiqc/multiqc_plots/pdf/mosdepth-cumcoverage-dist-id.pdf",
                "multiqc/multiqc_plots/pdf/samtools-stats-dp.pdf",
                "multiqc/multiqc_plots/pdf/samtools_alignment_plot-cnt.pdf",
                "multiqc/multiqc_plots/pdf/samtools_alignment_plot-pct.pdf",
                "multiqc/multiqc_plots/pdf/vcftools_tstv_by_count.pdf",
                "multiqc/multiqc_plots/pdf/vcftools_tstv_by_qual.pdf",
                "multiqc/multiqc_plots/png",
                "multiqc/multiqc_plots/png/bcftools-stats-subtypes-cnt.png",
                "multiqc/multiqc_plots/png/bcftools-stats-subtypes-pct.png",
                "multiqc/multiqc_plots/png/bcftools_stats_indel-lengths-cnt.png",
                "multiqc/multiqc_plots/png/bcftools_stats_indel-lengths-log.png",
                "multiqc/multiqc_plots/png/bcftools_stats_variant_depths.png",
                "multiqc/multiqc_plots/png/bcftools_stats_vqc_Count_Indels.png",
                "multiqc/multiqc_plots/png/bcftools_stats_vqc_Count_SNP.png",
                "multiqc/multiqc_plots/png/bcftools_stats_vqc_Count_Transitions.png",
                "multiqc/multiqc_plots/png/bcftools_stats_vqc_Count_Transversions.png",
                "multiqc/multiqc_plots/png/mosdepth-coverage-per-contig-single-cnt.png",
                "multiqc/multiqc_plots/png/mosdepth-coverage-per-contig-single-pct.png",
                "multiqc/multiqc_plots/png/mosdepth-cumcoverage-dist-id.png",
                "multiqc/multiqc_plots/png/samtools-stats-dp.png",
                "multiqc/multiqc_plots/png/samtools_alignment_plot-cnt.png",
                "multiqc/multiqc_plots/png/samtools_alignment_plot-pct.png",
                "multiqc/multiqc_plots/png/vcftools_tstv_by_count.png",
                "multiqc/multiqc_plots/png/vcftools_tstv_by_qual.png",
                "multiqc/multiqc_plots/svg",
                "multiqc/multiqc_plots/svg/bcftools-stats-subtypes-cnt.svg",
                "multiqc/multiqc_plots/svg/bcftools-stats-subtypes-pct.svg",
                "multiqc/multiqc_plots/svg/bcftools_stats_indel-lengths-cnt.svg",
                "multiqc/multiqc_plots/svg/bcftools_stats_indel-lengths-log.svg",
                "multiqc/multiqc_plots/svg/bcftools_stats_variant_depths.svg",
                "multiqc/multiqc_plots/svg/bcftools_stats_vqc_Count_Indels.svg",
                "multiqc/multiqc_plots/svg/bcftools_stats_vqc_Count_SNP.svg",
                "multiqc/multiqc_plots/svg/bcftools_stats_vqc_Count_Transitions.svg",
                "multiqc/multiqc_plots/svg/bcftools_stats_vqc_Count_Transversions.svg",
                "multiqc/multiqc_plots/svg/mosdepth-coverage-per-contig-single-cnt.svg",
                "multiqc/multiqc_plots/svg/mosdepth-coverage-per-contig-single-pct.svg",
                "multiqc/multiqc_plots/svg/mosdepth-cumcoverage-dist-id.svg",
                "multiqc/multiqc_plots/svg/samtools-stats-dp.svg",
                "multiqc/multiqc_plots/svg/samtools_alignment_plot-cnt.svg",
                "multiqc/multiqc_plots/svg/samtools_alignment_plot-pct.svg",
                "multiqc/multiqc_plots/svg/vcftools_tstv_by_count.svg",
                "multiqc/multiqc_plots/svg/vcftools_tstv_by_qual.svg",
                "multiqc/multiqc_report.html",
                "pipeline_info",
                "pipeline_info/nf_core_sarek_software_mqc_versions.yml",
                "reference",
                "reports",
                "reports/bcftools",
                "reports/bcftools/sentieon_dnascope",
                "reports/bcftools/sentieon_dnascope/joint_variant_calling",
                "reports/bcftools/sentieon_dnascope/joint_variant_calling/joint_germline.bcftools_stats.txt",
                "reports/mosdepth",
                "reports/mosdepth/test",
                "reports/mosdepth/test/test.recal.mosdepth.global.dist.txt",
                "reports/mosdepth/test/test.recal.mosdepth.region.dist.txt",
                "reports/mosdepth/test/test.recal.mosdepth.summary.txt",
                "reports/mosdepth/test/test.recal.regions.bed.gz",
                "reports/mosdepth/test/test.recal.regions.bed.gz.csi",
                "reports/samtools",
                "reports/samtools/test",
                "reports/samtools/test/test.recal.cram.stats",
                "reports/vcftools",
                "reports/vcftools/sentieon_dnascope",
                "reports/vcftools/sentieon_dnascope/joint_variant_calling",
                "reports/vcftools/sentieon_dnascope/joint_variant_calling/joint_germline.FILTER.summary",
                "reports/vcftools/sentieon_dnascope/joint_variant_calling/joint_germline.TsTv.count",
                "reports/vcftools/sentieon_dnascope/joint_variant_calling/joint_germline.TsTv.qual",
                "variant_calling",
                "variant_calling/sentieon_dnascope",
                "variant_calling/sentieon_dnascope/joint_variant_calling",
                "variant_calling/sentieon_dnascope/joint_variant_calling/joint_germline.vcf.gz",
                "variant_calling/sentieon_dnascope/joint_variant_calling/joint_germline.vcf.gz.tbi",
                "variant_calling/sentieon_dnascope/test",
                "variant_calling/sentieon_dnascope/test/test.dnascope.g.vcf.gz",
                "variant_calling/sentieon_dnascope/test/test.dnascope.g.vcf.gz.tbi"
            ],
            [
                "mosdepth-coverage-per-contig-single.txt:md5,115b9a93bcf511419e6622919172f321",
                "mosdepth-cumcoverage-dist-id.txt:md5,df2fc825dfc41e30f72f5d125a6447ee",
                "mosdepth_perchrom.txt:md5,115b9a93bcf511419e6622919172f321",
                "multiqc_citations.txt:md5,d40980f61eb64026d58102841b7f3860",
                "joint_germline.bcftools_stats.txt:md5,b66f04fbf0ca30785d07a4d0b4156bb7",
                "joint_germline.FILTER.summary:md5,b489034e69de07875cbf0f5548b4f55e",
                "joint_germline.TsTv.count:md5,803d74e40f7716202bae2a3a81c1ddfc"
            ],
            "No BAM files",
            "No CRAM files",
            [
                "joint_germline.vcf.gz:md5,ef572f2d4597e009fb10037b736b6b5b",
                "test.dnascope.g.vcf.gz:md5,2eec090f63df75523e2c0ecbc369a755"
            ]
        ],
        "meta": {
            "nf-test": "0.9.2",
            "nextflow": "25.04.2"
        },
        "timestamp": "2025-06-21T16:52:44.930404505"
    }
}